src/printapp/static/js/app.js

```diff
@@ -58,6 +58,14 @@ App.ApplicationController = Ember.Controller.extend({
       controller.set('loadingFailed', true);
       if (data.status === 504) {
         controller.set('uniflowIsDown', true);
+      } 
+      else if (data.status === 401) {
+        //Log out by posting to the logout api, which deletes the httponly cookie keeping the user logged in.
```

Comment:
Let's change this comment to: "The password in the session cookie is invalid, so sign the use out."